src/operator/nn/mkldnn/mkldnn_base.cc

```diff
@@ -392,6 +392,7 @@ mkldnn_memory_format_t GetDefaultFormat(const mkldnn::memory::desc &desc) {
       case mkldnn_gOhwi8o:
       case mkldnn_gOhwi16o:
       case mkldnn_gOhIw16o4i:
+      case mkldnn_Goihw16g_s8s8:
```

Comment:
What's this? It's first time for us to have this format?

Reply:
yes, when quantize s8s8 group conv.